### DISCUSSION TITLE
EASE-grid domains

### DISCUSSION THREAD

**Discussion by cbblanke** [NASA-LIS/LISF#1383] [Ideas]
I suggest allowing [EASE-grid] (https://nsidc.org/data/user-resources/help-center/guide-ease-grids) domains.  Would be nice to be able to assimilate satellite data (e.g. SMAP) on its native grid.

**Comment by dmocko** [comment]
Hi @cbblanke 

LIS already assimilates some EASE SMAP data (e.g., SMAP_L2_SM_P_NRT).

What products, in particular, are you hoping to use in LIS?

Also, there is support for running LDT/LIS directly on the EASE grid.  Select `"ease V2"` in your ldt.config for map projection:

`   character*50, public,  parameter :: LDT_easev2Id    = "ease V2"    !`

I see it's not in the LDT documentation, so I'm not sure about the other configs you will need to set.  See the code.

**Comment by cbblanke** [comment]
I was thinking of SMAP and SMAP-Sentinel soil moisture.  I was specifically asking about having the model grid match the EASE grid.  I will look at the code.  Thanks.
